recipes/opentelemac/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "opentelemac" %}
+{% set version = "v8p4r1" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # - url: https://gitlab.pam-retd.fr/otm/telemac-mascaret/-/archive/{{ version }}/telemac-mascaret-{{ version }}.tar.gz
+  - url: https://github.com/nicogodet/otm-sources/releases/download/{{ version }}/telemac-mascaret-{{ version }}.tar.gz
+    folder: opentelemac
+    # sha256: b48a0ee5cb43a68bc7255357544810ea67ea918a5ffd7fd61eeed41be4cf7b71
+    sha256: 4dc0fdf59c7befff277b51ef8bec1eaac9b4b12db071b8914739ea2156532ee5
+    patches:
+      - patches/f2py.patch  # [py>=39]
+
+build:
+  number: {{ build }}
+  skip: true  # [not linux]
+  script_env:
+    - TELEMAC_VERSION={{ version }}
+  missing_dso_whitelist:
+    - '*/libc.so.6'
+    - '*/libpthread.so.0'
+    - '*/libm.so.6'
+  ignore_run_exports:
+    - libstdcxx-ng
+    - numpy
+  rpaths:
+    - opentelemac/builds/gnu.dynamic/lib
+
+requirements:
+  build:
+    - cross-python_{{ target_platform }}    # [build_platform != target_platform]
+    - python                                # [build_platform != target_platform]
+    - numpy                                 # [build_platform != target_platform]
+  host:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - openmpi
+    - openmpi-mpifort
+    - python
+    - metis
+    - numpy
```

Comment:
shouldn't we need `sed`?
since we use it in `build.sh`?
In some environments, `sed` is not present by default

Reply:
Used at build time only

Reply:
You are probably relying on something in the docker image. If you want to be future proof you can add it here, on host, to make it available at build time. Just in case it is remove from the image.